bundles/org.openhab.voice.mimictts/src/main/resources/OH-INF/i18n/mimictts.properties

```diff
@@ -1,3 +1,10 @@
+# add-on
+
+addon.mimictts.name = Mimic Text-to-Speech
+addon.mimictts.description = The Mimic Text-to-Speech (TTS) service uses the offline open source Text-to-Speech engine designed by Mycroft A.I.
+
+# add-on
```

Comment:
Hello,
I think this second "# add-on" line is a typo ?

Reply:
It is generated by the plugin, and also exists for other files like `jsscripting.properties`.